modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java

```diff
@@ -68,14 +76,67 @@ public void deleteRelatedModel(
 				deletionType,
 				ObjectRelationshipConstants.DELETION_TYPE_CASCADE)) {
 
+			int maxpoolsize = GetterUtil.getInteger(
+				SystemProperties.get("system.executor.service.maxpoolsize"), 2);
```

Comment:
system.executor.service.maxpoolsize is a property used by SystemExecutorServiceUtil and used for startup, you should not use it here.